com.aobuchow.themes.moderndark/css/default_overrides.css

```diff
@@ -194,9 +194,19 @@ Section {
     background-color-titlebar: #17171b;
     background-color-gradient-titlebar: #17171b;
     border-color-titlebar: #17171b;
-    swt-titlebar-color: 'ColorDefinition#org-eclipse-ui-workbench-LINK_COLOR';
-	tb-toggle-hover-color: 'ColorDefinition#org-eclipse-ui-workbench-LINK_COLOR';
-	tb-toggle-color: 'ColorDefinition#org-eclipse-ui-workbench-LINK_COLOR';
+	/* There is a bug with using a color preference for the swt-titlebar-color */
+    swt-titlebar-color: #28ebc3;
+	/* some interactivity is lost here by making the hovers use the same color */
```

Comment:
Make a new bug about this? `Section { tb-toggle-hover-color: #... }` should be set to a different color than `swt-titlebar-color`